PlayerPanel.js

```diff
@@ -34,87 +34,121 @@ function update_pclist() {
 
 
 	});
-
-
-
+	const addPartyButtonContainer = $("<div class='add-party-container'></div>");
+	const addPartyButton = $("<button id='add-party'>ADD PARTY</button>");
+	addPartyButton.on('click', () => {
+		window.pcs.forEach(function (p, i) {
+			token_button({ target: $(`[data-name='${p.name}']`) }, i, window.pcs.length);
+		});
+	});
+	addPartyButtonContainer.append(addPartyButton);
+	pcs_list.append(addPartyButtonContainer);
 
 	var NEXT_COLOR = 0;
 	window.pcs.forEach(function(item, index) {
 
 		color = "#" + TOKEN_COLORS[NEXT_COLOR++];
 
 		pc = item;
-		row = $("<div/>");
-		row.css("padding", "20px");
-
-		player_name = $("<div/>");
-		player_name.text(pc.name).css("background", color).css("font-weight", "bold");
-		player_name.attr('data-nextcolor', NEXT_COLOR);
-		row.append(player_name);
-
-		row_inside = $("<div/>");
-		row_inside.css('border', '1px solid black');
-		row_inside.css("width", "100%");
-		row_inside.css("float", "left");
-		row.append(row_inside);
-
-		row_inside.append($("<div style='float:left;'><img width=60 height=60 src='" + pc.image + "'></div>"));
-
-		stats = $("<div/>");
-		stats.css("float", "left");
-
-		stats.append("<div><div><b>HP:</b></div><div class='hp'>-</div></div>");
-		stats.append("<div><div><b>AC:</b></div><div class='ac'>-</div></div>");
-		stats.append("<div><div><b>Passive Perception:</b></div><div class='pp'>-</div></div>");
-
-		row_inside.append(stats);
-
-		conditions = $("<div style='float:left;'/>");
-		conditions.append("<div><b>Conditions</b></div>");
-		conditions.append("<div class='conditions'></div>");
-
-		row_inside.append(conditions);
 
+		let playerData;

```

Comment:
Initialize this to an empty variably of the desired primitive. 

Personal preference. I'm not the maintainer, but there isn't a style guide.